app-tests/nek5000/nek5000_check.py

```diff
@@ -0,0 +1,48 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class nek5000_check(rfm.RegressionTest):
+    modules = ['Nek5000']
+    valid_systems = ['lumi']
+    valid_prog_environs = ['cpeGNU']
+    modules = ['partition/C']
+    executable = './nek5000'
+    strict_check = False
+    descr = (f'Nek500 test')
+    maintainers = ['']
+    build_system = 'CustomBuild'
+    #build_locally = False
+
+    @run_after('init')
+    def setup_run(self):
+        self.num_tasks = 32
+
+    @run_before('compile')
+    def build_test(self):
+        self.build_system.commands = [
+            'makenek bp5',
+        ]
+        self.prebuild_cmds = [
+            f'git clone --filter=blob:none --sparse https://github.com/Nek5000/NekExamples.git',
+            'cd NekExamples',
+            f'git sparse-checkout init --cone',
+            f'git sparse-checkout add bp5',
+            'cd bp5',
+        ]
+
+    @run_before('run')
+    def cwd_test(self):
+        self.prerun_cmds = [
+            'cd NekExamples/bp5',
+            'echo bp5 > SESSION.NAME && echo `pwd`"/" >> SESSION.NAME'
+        ]
+
+    @sanity_function
+    def assert_simulation_success(self):
+        return sn.assert_found(r'run successful',self.stdout)
+
+    @performance_function('s')
+    def time(self):
+        return sn.extractsingle(r'total elapsed time\s+[:]\s+(?P<wtime>\s+\S+)', # Fix scientific notation (E+) conversion
+                                self.stdout, 'wtime', float)
```

Comment:
@rsarm could you please help me with number conversion (scientific E10 base notation to float)?

Reply:
@mszpindler what's the format of the scientific notation value on the output?

Reply:
`total elapsed time             :   1.06893E+01 sec`

Reply:
It will automatically convert it to float from that format. You can try it with the regex
```python
r'total elapsed time \: (?P<wtime>\S+) sec'
```

Reply:
Fixed with ed84b54